### PR TITLE
Add missing `highlightingEnabled` - function

### PR DESCRIPTION
**Pull request by tim-gromeyer**
Useful to check if highlighting is enabled or not

### CODE REVIEW

**Comment by tim-gromeyer** [comment]
Huh, what happened here? A lot of builds fail because it fails to install Qt

**Comment by pbek** [comment]
Thanks a lot for your contribution!

> Huh, what happened here? A lot of builds fail because it fails to install Qt

Happens from time to time, I'll retry.

File: qmarkdowntextedit.cpp [hunk 1]
```diff
@@ -160,6 +160,14 @@ void QMarkdownTextEdit::setHighlightingEnabled(bool enabled) {
     }
 }
 
+/**
+ * @brief Returns whetever highlighting is enabled
```
Comment (pbek) [inline qmarkdowntextedit.cpp:164]: Can you please fix `whetever`, just use `if`.

File: qmarkdowntextedit.cpp [hunk 2]
```diff
@@ -160,6 +160,14 @@ void QMarkdownTextEdit::setHighlightingEnabled(bool enabled) {
...
+ * @return Returns true if highlighting is enabled, oterwise false
```
Comment (pbek) [inline qmarkdowntextedit.cpp:165]: I guess you meant `otherwise`, did you?

Comment (tim-gromeyer) [reply qmarkdowntextedit.cpp:165]: Oh, yes I did 😅 I'm from Germany and I'm still learning English...

It's time for me to install the Spellchecker plugin, I guess 😅

Comment (pbek) [reply qmarkdowntextedit.cpp:165]: CLion has that built-in. 😉 
Qt Creator is more "meh" in that regard...

In the browser, I like https://languagetool.org/ as an extension.

**Pull request merged**